AttitudeManager/Src/AM.cpp

```diff
@@ -2,19 +2,34 @@
 // Created by Anthony Luo on 2022-10-12.
 //
 #include "AM.hpp"
+#include "config.hpp"
 
 #include <array>
 #include <cstdlib>
 
+
+
 namespace AM {
 
+AttitudeManager::AttitudeManager(Flightmode* control_algorithm):
+    control_algorithm_(control_algorithm)
+{
+    for (uint8_t i{0}; i < config::NUM_MOTORS; i++) {
+        motorChannels_[i] = config::motors[i].driverConstructor();   
```

Comment:
I like that we initialize these in the constructor, but I would be curious if we might need to store them in a different structure so that they are easier to address, as per the previous comment.